pkg/sdk/databases.go

```diff
@@ -90,7 +90,14 @@ type databaseRow struct {
 func (row databaseRow) convert() *Database {
 	database := &Database{
 		CreatedOn: row.CreatedOn,
-		Name:      row.Name,
+	}
+	if row.Name != "" {
```

Comment:
I am not sure if I like this logic. Previously, we would always have the Name, now, we can end up with an empty identifier which is a big change. Maybe we should address SNOW-1561641 first? Let's chat on Monday.

Reply:
Reverted